### DISCUSSION TITLE
dateTransmission

### DISCUSSION THREAD

**Discussion by PhilippeLRETQC** [MTESSDev/FRW#102] [Annonces]
Bonjour,

La dateTransmission présente ici : https://github.com/MTESSDev/FRW/blob/main/Documentation/fichiers-transmission.md
Je voudrais confirmer que c'est bien la date de transmission par le client quand il a soumis le formulaire?

Merci beaucoup.

**Comment by vinnyboii** [comment]
Bonjour,
Oui c'est effectivement la date lorsque l'utilisateur a cliqué sur le bouton "Soumettre".
J'ai ajusté la documentation du paramètre ici: https://github.com/MTESSDev/FRW/edit/main/Documentation/fichiers-transmission.md
